tests/test_lint.py

```diff
@@ -336,3 +336,27 @@ def test_bug_number_pings(content, num_nits):
     assert len(nits) == num_nits
     if num_nits > 0:
         assert set(["BUG_NUMBER"]) == set(v.check_name for v in nits)
+
+
+def test_redundant_pings():
```

Comment:
The test above this uses `ping` as the ping name. Change that to `search` (line 317ff).

Reply:
Changed, the name of all the three above three test pings from 'ping' to 'search'